tests/helper/test-case.js

```diff
@@ -97,7 +97,6 @@ module.exports = {
 	 * @param {any} Prism The Prism instance which will tokenize `code`.
 	 * @param {string} code The code to tokenize.
 	 * @param {string} language The language id.
-	 * @returns {Array<string|Array<string|any[]>>}
```

Comment:
Do we still want a return type here?

Reply:
I think I removed it because TS can infer the type.

Reply:
I think this would be better to be explicit and leave it there. Then another developer wouldn't need to read the file to know what the return types are.

Reply:
Ok.

Reply:
Actually, this brings me right back to my question in #2071: How do we define a token stream?

Reply:
Can we leave the doc as-is & resolve it with that issue?

Reply:
Alright.